CrossRef-REST.js

```diff
@@ -0,0 +1,245 @@
+{
+	"translatorID": "0a61e167-de9a-4f93-a68a-628b48855909",
+	"translatorType": 8,
+	"label": "CrossRef REST",
+	"creator": "Martynas Bagdonas",
+	"target": "",
+	"minVersion": "5.0.0",
+	"maxVersion": null,
+	"priority": 90,
+	"inRepository": true,
+	"browserSupport": "gcsv",
+	"lastUpdated": "2018-04-11 10:00:01"
+}
+
+// Based on CrossRef.js (by Simon Kornblith), which uses OpenURL API
+
+// This translator uses the newer REST API
+// https://github.com/CrossRef/rest-api-doc
+// https://github.com/CrossRef/rest-api-doc/blob/master/api_format.md
+// REST API documentation not always reflect the actual API i.e. and
+// some fields are undocumented.
+// All CrossRef item types can be retrieved at http://api.crossref.org/types
+
+function fixAuthorCapitalization(string) {
```

Comment:
We should move this function to `Zotero.Utilities` because it occurs also in several other translators.